fe/fe-core/src/test/java/com/starrocks/sql/plan/AggregateTest.java

```diff
@@ -822,6 +823,34 @@ public void testWindowFunnel() throws Exception {
         FeConstants.runningUnitTest = false;
     }
 
+    @Test
+    public void testWindowFunnelWithInvalidDecimalWindow() throws Exception {
+        FeConstants.runningUnitTest = true;
```

Comment:
Why need to change the FeConstants.runningUnitTest  in the test?

Reply:
> Why need to change the FeConstants.runningUnitTest in the test?

It must be used to avoid failure of others uts.